src/test/js/shared/idam_helper.js

```diff
@@ -24,6 +24,9 @@ if (TestData.NOTIFY_API_KEY) {
   console.log("Notify client API key is not defined");
 }
 
+const URLSearchParams = require('url').URLSearchParams;
+var searchParams = new URLSearchParams();
```

Comment:
I just meant the const could be moved to the top - I guess the search params should be instantiated when needed, otherwise things might go weird if we try to reuse it in another method?